integrations/lambda/src/lambda_sample/data_generator.py

```diff
@@ -0,0 +1,2039 @@
+import random
+from datetime import datetime, timedelta
+
+class DataGenerator:
+    """
+    A list of dicts used to define the format of dimensions.
+
+    Format:
+    "name": str. Required. The name of the dimension.
+    "value_length": int. Optional. The length of the dimension value, when it is randomly generated. If neither this nor
+        "options" are provided, defaults to 10.
+    "random_options": [str]. Optional. An array of strings to pick at random as options for the dimension value.
+        Has precedence over "value_length". Values will be reused.
+    "unique_options": [str]. Optional. An array of strings to pick serially for the dimension value. Each value in
+        this array will be used once.
+
+    Example:
+    [
+        {
+            "name": "device_id",
+            "value_length": 9
+        },
+        {
+            "name": "region",
+            "random_options": ["us-east-1", "us-west-2"]
+        }
+    ]
+    """
+    dimension_templates: list
+
+    """
+    A list of dicts used to define the format of measures.
+
+    Format:
+    "name": str. Required. The name of the measure.
+    "type": str. Optional. The Timestream for LiveAnalytics data type of the measure. Valid options are "DOUBLE",
+        "BIGINT", "BOOLEAN", and "VARCHAR". Defaults to "DOUBLE".
+    "max_variation": Optional. The maximum amount a measure value can changed over time, positively or negatively.
+        For example, with a value of 5.0, measure values will increment by a max of 5.0 and a min of -5.0. Defaults to 1.5.
+    "max": Optional. The maximum measure value. Defaults to 100.0.
+    "min": Optional. The minimum measure value. Defaults to 0.0.
+    "random_options": Optional. A list of values the measure value can have. All elements of the list should be the same data type and
+        match the data type specified by the "type" field. This field overrides "max_variation", "max", and "min".
+        Values will be reused, in the same way as the dimension_templates "unique_options" field.
+    
+    Example:
+    [
+        {
+            "name": "temperature_celsius",
+            "type": "DOUBLE",
+            "max_variation": 2.0,
+            "max": 40.0,
+            "min": 30.0
+        },
+        {
+            "name": "symptoms",
+            "type": "VARCHAR",
+            "random_options": ["none", "headache", "shortness of breath", "fatigue", "nausea"]
+        }
+    ]
+    """
+    measure_templates: list
+
+    def __init__(self):
```

Comment:
(optional, for consistency)
```suggestion
    def __init__(self,  measure_templates: list, dimension_templates: list):
```
and set the below vars accordingly

Reply:
Added.